framework-plugins/lisk-framework-forger-plugin/src/controllers/forging_info.ts

```diff
@@ -12,50 +12,51 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { Request, Response, NextFunction } from 'express';
 import { BaseChannel, PluginCodec } from 'lisk-framework';
 import { KVStore } from '@liskhq/lisk-db';
 import { getForgerInfo } from '../db';
 import { Forger, DPoSAccountJSON } from '../types';
 
-export const getForgingInfo = (channel: BaseChannel, codec: PluginCodec, db: KVStore) => async (
-	_req: Request,
-	res: Response,
-	next: NextFunction,
-): Promise<void> => {
-	try {
-		const forgingDelegates = await channel.invoke<ReadonlyArray<Forger>>('app:getForgingStatus');
-		const encodedAccounts = await channel.invoke<string[]>('app:getAccounts', {
-			address: forgingDelegates.map(forger => forger.address),
-		});
-		const forgerAccounts = encodedAccounts.map(encodedAccount =>
-			codec.decodeAccount<DPoSAccountJSON>(encodedAccount),
-		);
+interface ForgerInfo extends Forger {
+	readonly username: string;
+	readonly totalReceivedFees: string;
+	readonly totalReceivedRewards: string;
+	readonly totalProducedBlocks: number;
+	readonly totalVotesReceived: string;
+	readonly consecutiveMissedBlocks: number;
+}
 
-		const data = [];
-		for (const forgerAccount of forgerAccounts) {
-			const forgerAddressBinary = Buffer.from(forgerAccount.address, 'hex').toString('binary');
-			const forgerInfo = await getForgerInfo(db, forgerAddressBinary);
-			const forger = forgingDelegates.find(aForger => aForger.address === forgerAccount.address);
+export const getForgingInfo = async (
+	channel: BaseChannel,
+	codec: PluginCodec,
+	db: KVStore,
+): Promise<ForgerInfo[]> => {
+	const forgingDelegates = await channel.invoke<ReadonlyArray<Forger>>('app:getForgingStatus');
+	const encodedAccounts = await channel.invoke<string[]>('app:getAccounts', {
+		address: forgingDelegates.map(forger => forger.address),
+	});
```

Comment:
As we already made `lisk-api-client` as dependency here, should not we use it? 

```js
import {createClient}  from '@liskhq/lisk-api-client`; 

const client = createClient(channel); 

const client.accounts.get();
```

Reply:
As discussed please create an issue to use `@liskhq/lisk-api-client` consistently across the plugins.

Reply:
Issue #6016  created to consistently use lisk-api-client in all the functional test. This will also be covered there.